webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/CheckInventoryUpdate.scala

```diff
@@ -72,7 +72,7 @@ class CheckInventoryUpdate(
       logger.trace("No update in node inventories main information detected")
     case false =>
       logger.info("Update in node inventories main information detected: triggering a policy generation") *>
-      IOResult.effectNonBlocking(asyncDeploymentAgent ! ManualStartDeployment(ModificationId(uuidGen.newUuid), RudderEventActor, "Main inventory information of at least one node were updated"))
+      IOResult.effectNonBlocking(asyncDeploymentAgent ! AutomaticStartDeployment(ModificationId(uuidGen.newUuid), RudderEventActor, "Main inventory information of at least one node were updated"))
```

Comment:
you need to remove comment as last parameter, `AutomaticStartDeployment` doesn't take them

Reply:
i'm amazed it compiled :/